esp.d.ts

```diff
@@ -31,13 +31,13 @@ export class Router {
     runAction<T>(modelId:string, action:(model : T) => void)  : void;
     getEventObservable<T, TEvent, TContext>(modelId:string, eventType:string, observationStage?: string) : EventObservable<T, TEvent, TContext>;
     getModelObservable<T>(modelId:string) : Observable<T>;
+    observeEventsOn(modelId: string, object : any, methodPrefix?: string) : Disposable;

```

Comment:
This probably makes sense being generic while we're in this space?

`observeEventsOn<TModel>(modelId: string, object: TModel, methodPrefix?: string) : Disposable;`